pyiron/lammps/base.py

```diff
@@ -573,25 +573,41 @@ def collect_output_log(self, file_name="log.lammps", cwd=None):
 
     def calc_minimize(
             self,
-            e_tol=0.0,
-            f_tol=1e-4,
+            ionic_energy_tolerance=0.0,
+            ionic_force_tolerance=1e-4,
+            e_tol=None,
+            f_tol=None,
             max_iter=1000000,
             pressure=None,
             n_print=100,
             style='cg'
     ):
         rotation_matrix = self._get_rotation_matrix(pressure=pressure)
         # Docstring set programmatically -- Ensure that changes to signature or defaults stay consistent!
+        if e_tol is not None:
+            warnings.warn(
+                "e_tol is deprecated as of vers. 0.3.0. It is not guaranteed to be in service in vers. 0.4.0. Use ionic_energy_tolerance instead.",
+                DeprecationWarning
+            )
+            ionic_energy_tolerance = e_tol
+            e_tol = None
+        if f_tol is not None:
+            warnings.warn(
+                "f_tol is deprecated as of vers. 0.3.0. It is not guaranteed to be in service in vers. 0.4.0. Use ionic_force_tolerance instead.",
+                DeprecationWarning
+            )
+            ionic_force_tolerance = f_tol
+            f_tol = None
         super(LammpsBase, self).calc_minimize(
```

Comment:
Here we should set both - e_tol, f_tol and ionic_energy_tolerance, ionic_force_tolerance so the warnings are raised from AtomisticGenericJob.